src/site/markdown/server/manage-guice-distributed-james.md

```diff
@@ -260,3 +261,70 @@ by setting the parameter `elasticsearch.index.mailbox.name` to the name of your
 re-creates index upon restart
 
 _Note_: keep in mind that reindexing can be a very long operation depending on the volume of mails you have stored.
+
+## Solving cassandra inconsistencies
+
+By choosing the cassandra, it means denormalization is the common modeling pattern to be using. 
+You may see some data is duplicated among tables. Especially, with tables having close relationships. 
+Some operations require multiple table writes, if there is any failure upon in the middle of an operation. 
+It could bring an inconsistent state to the data. The consequence could be dirty reads, unexpected failing writes.
```

Comment:
dirty reads is not what is described here, it's returning data that is in transient state (that should never appear outside of the system)